libfwupdplugin/fu-device.c

```diff
@@ -1871,6 +1871,12 @@ fu_device_set_quirk_kv(FuDevice *self, const gchar *key, const gchar *value, GEr
 			fu_device_add_guid(self, sections[i]);
 		return TRUE;
 	}
+	if (g_strcmp0(key, FU_QUIRKS_GUID "[quirk]") == 0) {
```

Comment:
I think this should be it's own `#define`, otherwise it's going to be harder to find in the codebase when looking for `Guid[quirk]`

Reply:
Fixed; I've move the docs commit to a future PR -- I think we need to merge the src/README.md into the existing fu-quirks.h docs -- it's weird having 90% the same in two places.